src/test/scala/io/delta/DeltaTableSuite.scala

```diff
@@ -37,6 +37,21 @@ class DeltaTableSuite extends QueryTest
     }
   }
 
+  test("forPath should not create a DeltaTable based on Non-Delta-Path") {
```

Comment:
nit: `forPath should not create a DeltaTable if a non-Delta path is provided`

Reply:
done.